views/pages/listaAcolhidos.ejs

```diff
@@ -1,15 +1,20 @@
 <%- contentFor('body') %>
-<section class="section">
-    <main class="main">
-        <h1 class="titulo-lista-acolhidos">Lista de Acolhidos</h1>
-        <ul class="lista-acolhidos">
-            <%for (acolhido of acolhidos){%>
-                <li>
-                    <a href="/acolhido/<%= acolhido.id %>">
-                        <%= acolhido.nome %>
-                    </a>
-                </li>
-            <%}%>
-        </ul>
-    </main>
-</section>
\ No newline at end of file
+    <section class="section">
+        <main class="main">
+            <h1 class="titulo-lista-acolhidos">Lista de Acolhidos</h1>
+            <ul class="lista-acolhidos">
+                <% for(var i=0; i< acolhidos.length; i++) { %>
```

Comment:
Por que a gente tá utilizando esse for ao invés do outro? o `for.. of` é bem mais descritivo e faz com que os objetos fiquem mais transparentes.

Reply:
feito